### PR TITLE
[loganalyzer] implementation with sample files

### PR DESCRIPTION
**Pull request by hrachya-mellanox**
* loganalyzer.py - log analyzer to analyze system log file and
additionally specified input files.

signed-off-by: hrachya@mellanox.com

### CODE REVIEW

File: ansible/roles/test/files/tools/loganalyzer.py [hunk 1]
```diff
@@ -0,0 +1,578 @@
...
+    ret_code = True
+    
+    if ((run_id is None) or (len(run_id) == 0)):
+        print 'ERROR: no run_id specified'
+        ret_code = False
+    
+    return ret_code
+#---------------------------------------------------------------------
+
+def check_logs(logs):
```
Comment (andriymoroz) [inline ansible/roles/test/files/tools/loganalyzer.py]: looks like this function is never used...

**Comment by stcheng** [comment]
could you update this pull request to compare it with the current master?

**Comment by taoyl-ms** [comment]
Could we have more document/comment on the meaning and usage of those sample files?

**Comment by hrachya-mellanox** [comment]
Hi Taoyu,
The usage of files is described in https://github.com/Azure/SONiC/wiki/LogAnalyzer#analysis-of-log-messages

**Comment by taoyl-ms** [comment]
I was talking about the log files in samples/sample1 folder. Shall we modify the sample string and runID there to be more meaningful? I'm feeling current sample difficult to follow

**Comment by hrachya-mellanox** [comment]
Hi Taoyu,
Following the description in the wiki page it should be clear I think, but if you'd like to suggest different content, you can send through mail and I can add it.

**Comment by taoyl-ms** [comment]
Hi Hrachyam,

What I mean is that we might be able to do some modification in the details to make the sample easier to read. Such as
-	In command_line.txt, modify the file paths to relative path in repo instead of /home/hrachya/projects/… so that user knows that those files can be found in the reop
-	Rename command_line.txt to README.txt or other obvious name so readers know this is the first file they need to read
-	Rename runID to something meaningful instead of myTest116
-	Write some descriptive message in ignore string sample instead of random strings like “67hello8"

**Comment by hrachya-mellanox** [comment]
i removed the sample directory, added sample to https://github.com/Azure/SONiC/wiki/LogAnalyzer. I'm also going to open new pull request for ansible files invoking loganalyzer which should help understanding its usage better.

**Pull request merged**